InvisibleCollectorLib/Model/Debt.cs

```diff
@@ -21,6 +21,7 @@ public class Debt : AttributesModel<Item>, IRoutableModel
         internal const string NetTotalName = "netTotal";
         internal const string PaidTotalName = "paidTotal";
         internal const string DebitTotalName = "dgereebitTotal";
```

Comment:
@pedromendes I don't know what this is but it probably on'w work on any scenario :P
```suggestion
        internal const string DebitTotalName = "debitTotal";
```